src/main/java/uk/gov/justice/hmpps/nomis/datacompliance/repository/jpa/model/DuplicateOffender.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.hmpps.nomis.datacompliance.repository.jpa.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "OFFENDERS")
+public class DuplicateOffender {
```

Comment:
I suspect you can do the query below without defining this table entity (assuming there is a OFFENDER jpa model already)